scripts/variant-kernel/README.md

```diff
@@ -0,0 +1,39 @@
+# Script: Variant Kernel Information
+
+This script is used to generate the data for `[lang]/os/[version]]/version-information/variants/`. You will need to run it on ever minor version.
```

Comment:
nit: is there an extra right bracket after `version`? `]`

Reply:
Also `ever` -> `every`?

Reply:
Good catches!